scaffolds/typescript/ts-node-env

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+docker run --rm \
+  -u $(id -u):$(id -g) \
+  -v $(pwd):/tmp/project:rw \
+  -w /tmp/project \
+  sandrokeil/typescript $@
```

Comment:
I'd rather we used a version tag here, otherwise we risk the image changing unexpectedly.

May I ask the added value of not using a Node version directly, given we have to install dependencies anyway ?

Reply:
>I'd rather we used a version tag here, otherwise we risk the image changing unexpectedly.

The image didn't have a version tag.

>May I ask the added value of not using a Node version directly, given we have to install dependencies anyway ?

None. I don't know why I wanted to use a typescript image.